More misc fixups

GiudGiud opened the pull request.

GiudGiud committed:
Fixup link in FVTimeKernel doc, refs #15968
Fix listing to actually include a genericConstantMaterial in its documentation

GiudGiud committed:
Add test for linear nodal constraint in framework

GiudGiud committed:
Remove ugly looking deprecation warning on new app creation, refs #17316

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1060869/) on bece315 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/21173/site)

This comment will be updated on new commits.

lindsayad commented:
I am not all that surprised that there are still some issues with distributed mesh for the linear nodal constraint. Let me know if you want me to look into that

lindsayad commented:
Actually I'm wondering if you still get the exodiffs with distributed mesh if you run with `allow_renumbering = false`? For an object that relies on raw node IDs we should probably just error if allow renumbering is anything but false

GiudGiud commented:
That's a good idea. I think I need to do a general review of this behavior.

GiudGiud commented:
looks like that fixed the nodal linear constraint, but other nodal constraints are still affected. Well it's a good news though overall.

lindsayad commented:
> other nodal constraints are still affected

affected by what? Running with a distributed mesh?

GiudGiud committed:
Fix replicated mesh checks, refs #21185

GiudGiud committed:
Allow a warning for parsed generation sideset, because it s so useful
refs #21185

GiudGiud committed:
Allow for same number of columns as number of properties in readpropertyUO (from CSV) refs #19109

GiudGiud committed:
Fix viscosity term in porous media equation, refs #16756

Fixup PINSFV docs
refs #16756

Fixup convection correlation equation format for FVIks for it

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1067717/) on 74c2254 wanted to post the following:

Your code requires style changes.  

A patch was auto generated and copied [here](https://mooseframework.inl.gov/docs/PRs/21173/clang_format/style.patch)  
You can directly apply the patch by running, in the top level of your repository:

`curl -s https://mooseframework.inl.gov/docs/PRs/21173/clang_format/style.patch | git apply -v`  

Alternatively, with your repository up to date and in the top level of your repository:

`git clang-format 8888f041c177b7718279cbb4c929a82a8d548ce6`

GiudGiud committed:
Add parameter groups to sort very long list of optional parameters, refs #20535

GiudGiud commented:
@dschwen thoughts about the parsed parameters renaming?

GiudGiud commented:
Should be ready for review.
I broke off the controversial stuff in another PR

lindsayad commented on modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md:
```diff
@@ -5,24 +5,16 @@ equations. This kernel must be used with the canonical PINSFV variable set,
 \end{equation}
 where $F_i$ is the i-th component of the friction force, f the friction factor, which may be anisotropic,
-$\epsilon$ the porosity and $\rho$ the fluid density and $v_i$ the fluid superficial velocity.
+$\epsilon$ the porosity and $\rho$ the fluid density and $v_d$ the fluid superficial velocity.
```
I think that the $i$ index was meant to refer to the velocity component and you're retaining that on the force and coefficient. Seems like it might be worth keeping around on the velocity as well? Maybe $v_{d,i}$?

GiudGiud replied:
ah good call. I ll fix it now

GiudGiud commented on framework/src/outputs/AdvancedOutput.C:
```diff
@@ -700,30 +704,30 @@ AdvancedOutput::addValidParams(InputParameters & params, const MultiMooseEnum &
     params.addParam<ExecFlagEnum>(
         "execute_postprocessors_on", empty_execute_on, "Control of when postprocessors are output");
-    params.addParamNamesToGroup("execute_postprocessors_on", "Variables");
+    params.addParamNamesToGroup("execute_postprocessors_on", "Selection/restriction of output");
```
it's not a file name, it s what appears in the parameter section
for example here:
https://mooseframework.inl.gov/source/multiapps/CentroidMultiApp.html

lindsayad replied:
Nice, I like that then

GiudGiud commented:
should finally pass now. 
would have been faster to fix those sideset generators than allow the warnings ..

loganharbour commented:
Refs #21185

@GiudGiud this probably closes the above. Make sure to try to keep track of issues with these combined PRs...

You also broke mockingbird 😢

loganharbour commented:
SideSetsFromNormalsGenerator does work with distributed mesh as long as it doesn't sweep. The direct error is incorrect here

loganharbour commented:
Refs #19109 #16756 #15968 #17316 #20535
